config/locales/en.yml

```diff
@@ -1,3 +1,243 @@
 en:
```

Comment:
why didn't these translations exist before?

Reply:
Because we never had to use Rails time localization for anything. The date control was front-end only. This is backend.